common-lib/src/cmr/common/api/web_server.clj

```diff
@@ -41,10 +41,10 @@
   ONE_MB)
 
 (def MAX_REQUEST_BODY_SIZE
- "The maximum size of a request to body. This is set to 500KB prevent large requests coming in
-  that cause out of memory exceptions. A large ISO document like AST_L1A can be 120K characters which
-  would take up about 240KB in Java. This value is set to 5 MB."
- (* 5 ONE_MB))
+ "The maximum request body size which the application will accept. This is set to prevent large,
```

Comment:
You can search Splunk to see the most memory any of our apps use:
sourcetype=*cmr*-prod logjvmstatisticsjob "Used memory" | rex "Used memory.*\[(?<used>\d+)\] Percent" | stats max(used) by sourcetype

Search, indexer, and metadata-db set the max heap size to 5GB. The other apps use the defaults. The default heap size is 1/4 of memory on the machine and our smallest instance type is m3.medium with 3.75 GB of RAM or roughly 940 MB. You can see that none of the apps are within 50MB of hitting the max heap size. Even if they were garbage collection would take place, and so an OOM from a 50MB request is really unlikely to ever happen (never is not in my vocabulary).